Kibol-Alert/ViewModels/ClubVM.cs

```diff
@@ -0,0 +1,19 @@
+﻿using Kibol_Alert.Models;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Kibol_Alert.ViewModels
+{
+    public class ClubVM
+    {
+        public int Id { get; set; }
+        public string Name { get; set; }
+        public string League { get; set; }
+        public string LogoUri { get; set; }
+        public virtual ICollection<ClubRelations> ClubRelations { get; set; }
+        public virtual ICollection<User> Fans { get; set; }
```

Comment:
We don't want to have full info about users, we need just VM of them. So, change User to UserVM. ClubRelations is fine